crates/iota/src/validator_commands.rs

```diff
@@ -700,21 +681,21 @@ impl IotaValidatorCommand {
                     "pending stake",
                 ]);
 
-                for IotaValidatorSummary {
-                    iota_address,
-                    name,
-                    staking_pool_iota_balance,
-                    pending_stake,
-                    ..
-                } in active_validators
-                {
-                    builder.push_record([
-                        iota_address.to_string(),
-                        name,
-                        staking_pool_iota_balance.to_string(),
-                        pending_stake.to_string(),
-                    ]);
-                }
+                let client = context.get_client().await?;
+
+                client
+                    .governance_api()
+                    .get_latest_iota_system_state()
+                    .await?
+                    .iter_committee_members()
```

Comment:
Should `IotaValidatorCommand::List` command list "active validators" or "committee members"?

Reply:
It should list the active validators, we'll display if they are in the committee in another PR https://github.com/iotaledger/iota/issues/6052

Reply:
Thanks both, modified it in [2021666](https://github.com/iotaledger/iota/pull/6015/commits/20216661e9b63e8e26f8a3e4e3461635059b1279).